### DISCUSSION TITLE
Should Fritter worried?

### DISCUSSION THREAD

**Discussion by ghost** [jonjomckay/fritter#758] [Q&A]
![image](https://github.com/jonjomckay/fritter/assets/123662124/2c573fdf-ea68-49a3-b0ac-b9e8018ef07b)
![image (1)](https://github.com/jonjomckay/fritter/assets/123662124/76c98636-89e5-4375-b5ed-3ce1411f8cc7)

@jonjomckay

**Comment by papjul** [comment]
Most likely story:
1) Twitter didn't pay the bill of a contract ending on June 30rd https://www.platformer.news/p/twitter-stiffs-google
2) Musk asked to rate-limit the platform
3) Twitter is DDOSing itself https://sfba.social/@sysop408/110639435788921057

Nothing to do with scrapers IMO.

Plus, the first step for them would be to report the repo to GitHub, you don't go to court without trying that first.

There are many scrappers for social networks or YouTube on GitHub, the only one I remember having a report was yt-dl but it only lasted a few days before it was back. So I would say there is nothing much to worry at this point.

**Reply by gyreas** [reply]
Could this be why I get "This functionality is no longer supported by Twitter" error when I try to open trending topics?

**Reply by ghost** [reply]
No, that's a choice made by @jonjomckay

**Comment by rsssxd** [comment]
@TheHCJ 
Will the Quacker project go on?

Twitter has lifted the ban on viewing.

**Reply by j-fbriere** [reply]
[Squawker](https://github.com/j-fbriere/squawker) is the fork of Quacker that I'm maintaining.

Right now the ban on viewing has been only partly lifted.

The feeds are still not working. But the rest is working on version [3.0.1](https://github.com/j-fbriere/squawker/releases/tag/v3.0.1).

The feed and group pages are simply unselected by default because of the current Twitter situation. 
But you can reactivate them at any moment, hopefully when the feed ban is lifted.

**Reply by ghost** [reply]
The feeds relies on Tweet searching which is currently broken.
